### PR TITLE
chokidar: Change event type to `string`

### PR DESCRIPTION
**Pull request by ghost**
Please fill in this template.

- [X] Make your PR against the `master` branch.
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [X] Run `npm run lint -- package-name` if a `tslint.json` is present.

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/paulmillr/chokidar/blob/master/lib/nodefs-handler.js#L26
- [X] Increase the version number in the header if appropriate. (N/A)

Tested this with:
```js
const f = new (require("chokidar").FSWatcher)();
f.add("a.js");
f.on("raw", (event, path, details) => {
    console.log(event);
});
```
It outputs a string `"change"`. Might also be nice to change the type to a union if all possible events can be known.

### CODE REVIEW

File: chokidar/chokidar-tests.ts [hunk 1]
```diff
@@ -36,6 +41,7 @@ watcher.unwatch('new-file*');
 watcher.close();
 
 // One-liner
-require('chokidar').watch('.', {ignored: /[\/\\]\./}).on('all', function(event:string, path:string) {
-    console.log(event, path);
+chokidar.watch('.', {ignored: /[\/\\]\./}).on('all', (event, path) => {
```
Comment (andy-ms) [inline chokidar/chokidar-tests.ts]: `require('chokidar')` isn't type-checked. Had to add an `"all"` overload to fix this test.

**Review by TimvdLippe — APPROVED** [review]
Yes this seems reasonable to me. Thanks for fixing!

**Comment by vvakame** [comment]
We can't get response from author. please ping me after 3 days. I'll merge this PR.

**Comment by reppners** [comment]
👍

**Pull request merged**